.husky/pre-commit

```diff
@@ -2,3 +2,6 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npx lint-staged --allow-empty
+
+npm run docs:generate
+git add ./docs-v2/snippets/generated/
```

Comment:
can we just run it when stuff has changed inside docs-v2?

Reply:
The task is what changes stuff inside of `docs-v2`.

Reply:
not sure I get what you mean

Reply:
Nothing has changed inside `docs-v2` until we run the command

Reply:
If I read the code correctly: it changes stuff if flows.yaml is modified (should be done in the other repo), a `provider:` has been changed, so it's a change in docs-v2. Am I missing something?